extension.js

```diff
@@ -151,23 +156,24 @@ function showSplashScreens() {
             width: monitor.width,
             height: monitor.height,
         });
+        Main.layoutManager.addChrome(splashscreen);
         splashScreens.push(splashscreen);
     });
-    GLib.timeout_add(GLib.PRIORITY_DEFAULT, 4000, () => {
-        hideSplashScreens();
-    });
```

Comment:
This was meant for debugging purpose if you have an error between the extension enable process the splashcreen would never be hided.